### ISSUE TITLE
New unsaved files throw error when trying to print

### ISSUE THREAD

**Issue by baincd** [PeterWone/vsc-print#88]
New files that have not been saved throw an error when trying to print

![image](https://user-images.githubusercontent.com/5075659/112917025-d02a5f80-90cf-11eb-8c3d-2946eabbbf2d.png)

![image](https://user-images.githubusercontent.com/5075659/112917052-e0dad580-90cf-11eb-9819-f5a32be924f0.png)

**Comment by baincd** [comment]
I am researching this issue

**Comment by PeterWone** [comment]
I thought I'd already fixed that one. It has to do with the fact that there's no corresponding file on disk. What is _supposed_ to happen is pull the buffer from the active editor.

**Comment by baincd** [comment]
The issue seems to be with some of the print folder functionality.  I had to add checks in 2 places to prevent errors for files not on disk.

I also added a check when using the print folder command and the current editor is a file not on disk.  In this scenario, what was happening was the extension attempted to print all the files from the user's home directory, which I didn't think was expected or desired.
